source/documentationUtils.py

```diff
@@ -75,3 +78,20 @@ def reportNoDocumentation(fileName: str, useMsgBox: bool = False) -> None:
 		)
 	else:
 		queueHandler.queueFunction(queueHandler.eventQueue, ui.message, noDocMessage)
+
+
+@lru_cache(maxsize=1)
+def _getSanitizedHtmlLicense() -> str:
+	licenseFilename: str = getDocFilePath("copying.txt", False)
+	with open(licenseFilename, "r", encoding="utf-8") as licenseFile:
+		htmlLicense = markdown.markdown(licenseFile.read())
+	return nh3.clean(htmlLicense)
```

Comment:
Do we need to sanitize here? I think that this function is only called in `ui.browseableMessage` and sanitization will be required by #17581 in any case.

Couldn't we rather block the current PR until #17581 is merged?

Reply:
I think it's safer to sanitize it here, in case we use a different wx control (such as HtmlWindow) to display it.
We also cache the cleaning, so after #17581, we could skip the sanitization there.

Reply:
This now the case with `WebView` being used